nat-lab/tests/telio.py

```diff
@@ -1211,6 +1213,83 @@ async def probe_pmtu(self, host: str, expected: int):
         await self._write_command(["pmtu", host])
         await ev.wait()
 
+    async def get_coredump_pattern(self) -> tuple[str, str]:
+        process = await self._connection.create_process(
+            ["sysctl", "kernel.core_pattern"]
+        ).execute()
+        # sysctl returns a pattern that can contain just a file name, or with absolute path, like
+        # `kernel.core_pattern = /var/crash/core-%h-%e-%s-%p` or simply `kernel.core_pattern = core`
+        core_pattern = process.get_stdout().split("= ", 1)[1].rsplit("/", 1)
+
+        path = "."
+        # if there is only one element it means that there is no path present in pattern
+        if len(core_pattern) > 1:
+            path = core_pattern[0]
+        # remove the filename flags 'core-%h-%e-%s-%p'
+        file_name = core_pattern[-1].split("%", 1)[0]
+
+        return path, file_name
+
+    async def clear_core_dumps(self):
+        if (os.environ.get("NATLAB_SAVE_LOGS") is None) or (
+            self._connection.target_os != TargetOS.Linux
+        ):
```

Comment:
Nit: I'm kinda surprised black doesn't complain about it but I think the unnecessary parentheses here makes the conditional more complicated to read

Reply:
The alternative that `black` suggests is 
```python
        if (
            os.environ.get("NATLAB_SAVE_LOGS") is None
            or self._connection.target_os != TargetOS.Linux
        ):
```

Which is slightly more readable